### ISSUE TITLE
[AWS Power Up] Asset price Prediction Using QNN & Classical Hybrid Neural Network in Pennylane, Tensorflow & Amazon Braket

### ISSUE THREAD

**Issue by pratjz** [XanaduAI/QHack2022#58]
### Team Name: 

Pratjz
### Project Description: 

In this project, we will use Quantum Neural Network along with classical layers to create Hybrid Neural network using Pennylane & Tensorflow & Amazon Braket platform to present the state of the art of quantum algorithms for Financial applications, with particular focus to use cases on Continuous variable prediction problems for example Asset price prediction. We will use Boston housing data taken from the StatLib library maintained at Carnegie Mellon University, in the process we will employ various qml & ml techniques along with custom ansatz, The Ultimate goal is to Find out & demonstrate quantum advantage, be it in terms of dimensionality, accuracy, time complexity specially focusing on Scope of Quantum computing in Finance which is one of the most sought after area in Quantum computing second to Quantum Chemical simulations which itself branches out into numerous fields

Quantum Machine Learning is an emerging field that promises to solve intractable problems. There are multiple quantum machine learning algorithms that have emerged from their classical counterparts. Specially the so-called Quantum Neural Networks.

The aim of this project is to develop a clear understanding of the promises and limitations of the current state-of-the-art of quantum algorithms for machine learning, also to define directions for future research in this exciting field. And then draw the parallels to recent ‘quantum-inspired’ results, and explain the implications of these results for quantum machine learning applications. Looking for areas which might bear larger advantages for QML algorithms, like Finance & Quantum Chemical simulations

### Source code: 

https://github.com/pratjz/Qhack2022

### Resource Estimate: 

To run experiments with multiple Iterations on actual QPU like Rigetti's Aspen series & IonQ & Simulators like SN1, TN1 from Amazon Braket we roughly estimate  atleast 1000$ or little more, We will be needing to run multiple iterations & shots with different algorithmic settings & parameters & different ansatz along with multiple optimizers to find good accuracy & compare / benchmark

### Challenges
Amazon Braket Challenge
Quantum Finance Challenge

### References
- Maria Schuld and Francesco Petruccione. Supervised Learning with Quantum Computers.
- Boston Housing – Price Prediction https://rstudio-
- Pennylane.ai documentation https://pennylane.readthedocs.io/en/stable/
- Quantum Machine Learning for Finance https://arxiv.org/pdf/2109.04298.pdf
- Continuous-variable quantum neural networks https://arxiv.org/abs/1806.0687

**Comment by isaacdevlugt** [comment]
Thank you for your Power Up submission! As a reminder, the final deadline for your project is February 25 at 17h00 EST. Submissions should be done here: https://github.com/XanaduAI/QHack/issues/new?assignees=&labels=&template=open_hackathon.md&title=%5BENTRY%5D+Your+Project+Title

This issue will be closed shortly.

Good luck!
